example/lib/main.dart

```diff
@@ -179,6 +198,19 @@ class MyHomePage extends StatelessWidget {
           ),
         ),
       ),
+      floatingActionButton: MaterialButton(
```

Comment:
A floating action button could look a little sloppy, but I figured this should be fine?